File: CS/T191208/ISupportCustomizationDrag.cs
using DevExpress.XtraBars;
using System;
using System.Collections.Generic;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace T191208
{
    public interface ISupportCustomizationDrag
    {
        void AddMouseMoveHandler(MouseEventHandler mouseMove);
        void AddMouseDownHandler(MouseEventHandler mouseDown);
        void AddMouseUpHandler(MouseEventHandler mouseUp);
        BarItem GetDraggingItem(Point location);
        Control GetControlToPerformDragging(Point location);
        void InitControl(BarManager manager);
    }
}

File: VB/T191208/ISupportCustomizationDrag.vb
Imports DevExpress.XtraBars
Imports System
Imports System.Collections.Generic
Imports System.Drawing
Imports System.Linq
Imports System.Text
Imports System.Threading.Tasks
Imports System.Windows.Forms

Namespace T191208
    Public Interface ISupportCustomizationDrag
        Sub AddMouseMoveHandler(ByVal mouseMove As MouseEventHandler)
        Sub AddMouseDownHandler(ByVal mouseDown As MouseEventHandler)
        Sub AddMouseUpHandler(ByVal mouseUp As MouseEventHandler)
        Function GetDraggingItem(ByVal location As Point) As BarItem
        Function GetControlToPerformDragging(ByVal location As Point) As Control
        Sub InitControl(ByVal manager As BarManager)
    End Interface
End Namespace
